spec/support/acts_as_matcher.rb

```diff
@@ -0,0 +1,45 @@
+require 'rspec/expectations'
```

Comment:
Yes, it was added by jason. I renamed it from `act_as_matcher` to `acts_as_matcher` but didn't notice the duplication during rebasing. Deleted the other file.